lib/mumuki/domain/organization/settings.rb

```diff
@@ -24,4 +28,12 @@ def login_methods
   def forum_discussions_minimal_role
     (@forum_discussions_minimal_role || 'student').to_sym
   end
+
+  def archived?
+    archiving_enabled? && archive_from.to_datetime < DateTime.now
+  end
+
```

Comment:
Does it make sense organization to be archived and disabled?

I think we are missing something like a global state of organization.

Reply:
We'll introduce in a new one the semantics related to `in_preparation?` (not defined yet :stuck_out_tongue: ).
Maybe there it'll make sense to reify them in 3 states.
Pre -> 403 and "The course will start X"
200
Post -> 410 and "The course has finished".

Do you another possible state?

Reply:
Yup, that sound goods. I'd introduce an `organization.status` with three possible values.